pom.xml

```diff
@@ -15,8 +15,8 @@
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Job+Restrictions+Plugin</url>
 
     <properties>
-        <jenkins.version>1.609.3</jenkins.version>
-        <java.level>6</java.level>
+        <jenkins.version>2.60.3</jenkins.version>
```

Comment:
The baseline update to this version is not going to happen now. Something like 1.651.x or 2.46.x could be acceptable